dpgen2/exploration/report/report_trust_levels_spin.py

```
@@ -0,0 +1,300 @@
+        Returns
+        -------
+        cand_frames   List[Tuple[int,int]]
+            Candidate frames. A list of tuples: [(traj_idx, frame_idx), ...]
+        """
+        self.traj_cand_picked = []
+        for tidx, tt in enumerate(self.traj_cand):
+            for ff in tt:
+                self.traj_cand_picked.append((tidx, ff))
+        if max_nframes is not None and max_nframes < len(self.traj_cand_picked):
+            # select by max magnetic force
+            max_devi_af = self.model_devi.get(DeviManagerSpin.MAX_DEVI_AF)
```

Ruff

Local variable `max_devi_af` is assigned to but never used

Remove assignment to unused variable `max_devi_af`

(F841)